Cargo.toml

```diff
@@ -18,11 +20,11 @@ harness = false
 [dependencies]
 criterion = { version = "0.3.5", features = ["html_reports"] }
 merlin = "3.0.0"
-ark-ec ="^0.3.0"
-ark-ff ="^0.3.0"
-ark-bls12-381 = "^0.3.0"
-ark-std ="^0.3.0"
-ark-serialize = { version = "^0.3.0", features = ["derive"] }
+ark-ec = "^0.4.0"
```

Comment:
Carot is generally good when in development, once you get to a stable release, you may want to fix the version so you don't get rugged